django-backend/fecfiler/utils.py

```diff
@@ -19,8 +19,11 @@ def custom_exception_handler(exc, context):
     if response is not None and response.status_code == 403:
         delete_user_logged_in_cookies(response)
 
-    # Do not allow an error response body
-    # if getattr(response, 'data'):
-    #     response.data = None
+    # Do not allow an error response body unless validation
+    data = getattr(response, 'data')
+    exception_type = type(exc)
+    print(data)
```

Comment:
@sasha-dresden I think we should avoid using `print` statements

Reply:
Good point. Switched it over to use structlog